manifests/concourse-manifest/concourse-base.yml

```diff
@@ -45,7 +45,6 @@ resource_pools:
       ephemeral_disk:
         size: 102400
         type: gp2
-      key_name: (( grab terraform_outputs_concourse_key_pair_name ))
```

Comment:
Why are you removing this?

Reply:
Because this is reverting to use the default key. The bosh manifest specifies a default key, and that one gets used if a key_name isn't specified at the instance_group level like this. Removing this override makes it use that default.